### DISCUSSION TITLE
What data is already available in the DRB?

### DISCUSSION THREAD

**Discussion by jsadler2** [USGS-R/drb-do-ml#4] [General]
There has already been done quite a bit of data gathering in the DRB by the temperature project and others. I would like to use this thread to document some of those resources that are already available to us.

**Comment by jsadler2** [comment]
Data categories include:
1. Meteorological driver data #5 
2. Static features data #6 
3. Network connectivity data (distance matrices)
4. Discharge and water temperature data

**Reply by lekoenig** [reply]
To start to flesh out the static features category, I'll add channel slope and land use to the wish list. 

For the DRB, most recent NLCD land use is 2019 and EPA StreamCat-processed NLCD is available for 2011. Land use will also be used for the inland salinity project. Channel slope is available with the NHDPlusV2 flowlines, although we would have to crosswalk NHD to the PRMS polylines. @jsadler2 Do you know if ongoing projects have developed existing tools or scripts for mapping NHD reaches to the PRMS framework?

**Reply by jsadler2** [reply]
Good question about mapping the NHD to PRMS. This is something @jds485 and I were talking about the other day. It would open up a lot of attributes. I'll ask around about this.

**Reply by jsadler2** [reply]
@lekoenig - HRU slope and channel slope are part of the PRMS attributes. https://github.com/USGS-R/drb-do-ml/issues/6#issuecomment-952139548.
